packages/renderer/src/lib/appearance/Appearance.svelte

```diff
@@ -1,18 +1,20 @@
 <script lang="ts">
 import { onDestroy, onMount } from 'svelte';
+import { get } from 'svelte/store';
 
 import { AppearanceSettings } from '../../../../main/src/plugin/appearance-settings';
+import { isDark } from '../../stores/appearance';
 import { onDidChangeConfiguration } from '../../stores/configurationProperties';
-import { AppearanceUtil } from './appearance-util';
 
 const APPEARANCE_CONFIGURATION_KEY = AppearanceSettings.SectionName + '.' + AppearanceSettings.Appearance;
 async function updateAppearance(): Promise<void> {
   const html = document.documentElement;
 
+  // Wait for isDark to get updated
+  await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
I don't think this should be used here. You should find a way to react on `$isDark` change

Reply:
This will get called when is the appearance changed, but the `isDark` will return previous value, since it doesn't have enough time to change its value. 

With this change if you e.g. change from light->dark the `isDark` will be before line 14 e.g. `false`, and after line `14` it will be `true`. Withou this, the `get(isDark)` would return `false`